django-stubs/core/files/storage.pyi

```diff
@@ -43,4 +43,4 @@ class DefaultStorage(LazyObject): ...
 
 default_storage: Any
 
-def get_storage_class(import_path: Optional[str] = None) -> Type[Any]: ...
+def get_storage_class(import_path: Optional[str] = None) -> Type[Storage]: ...
```

Comment:
I am pretty sure that this notation will disallow subclassing. I guess it is better to write

```
_StorageType = TypeVar('_StorageType', bound=Storage)

def get_storage_class(import_path: Optional[str] = None) -> Type[_StorageType]: 
```

Reply:
I could very well be wrong, but I think it does allow subclassing.  This works:

```
class A:
    pass

class B(A):
    pass

class C(B):
    pass

def lol() -> Type[A]:
    return C
```

Reply:
Extending the above example:

```
def lol2() -> Type[B]:
    return A # error: Incompatible return value type (got "Type[A]", expected "Type[B]")
```

Reply:
Ok, looks good for me. Sorry for the extra work. 👍

Reply:
no problem!  Happy to help.